.gitignore

```diff
@@ -1,3 +1,13 @@
 *.pyc
 lib/*.pyc
 .idea
+tests/bin/*
+tests/lib/*
+tests/include/*
+tests/.Python
+.cache
+bin/*
+lib/*
```

Comment:
I am willing to merge this. But i don't like idea of puting lib/* in .gitignore as we use this directory and it will lead to confusion. do you really need to ignore everything in that directory?

Reply:
Ahh; I added that directory because that's where the virtualenv stores installed python dependencies.  Let me have a look at this again over the next few days to see if I can update this to move that into a different spot.  This PR is also a little old now and I'll try to make some time to update it to the current standards.

Reply:
ok. thanks!